src/qt/pivx/dashboardwidget.cpp

```diff
@@ -212,15 +211,11 @@ void DashboardWidget::loadWalletModel()
         // Notification pop-up for new transaction
         connect(txModel, &TransactionTableModel::rowsInserted, this, &DashboardWidget::processNewTransaction);
 #ifdef USE_QTCHARTS
-        // chart filter
-        stakesFilter = new TransactionFilterProxy();
-        stakesFilter->setDynamicSortFilter(true);
-        stakesFilter->setOnlyStakes(true);
-        stakesFilter->setSourceModel(txModel);
-        hasStakes = stakesFilter->rowCount() > 0;
-
-        onHideChartsChanged(walletModel->getOptionsModel()->isHideCharts());
-        connect(walletModel->getOptionsModel(), &OptionsModel::hideChartsChanged, this, &DashboardWidget::onHideChartsChanged);
+        if (!walletModel->getOptionsModel()->isHideCharts()) {
+            onHideChartsChanged(false);
+            connect(walletModel->getOptionsModel(), &OptionsModel::hideChartsChanged, this,
+                    &DashboardWidget::onHideChartsChanged);
+        }
```

Comment:
with this change it is impossible to show the charts back if you start with `--hidecharts` as there is no connection.
Also the charts aren't actually hidden (since we don't call `ui->right->setVisible(fShowCharts)` anymore).
The empty chart is shown instead.